app/controllers/v1/facilities/ccp_controller.rb

```diff
@@ -67,8 +67,15 @@ def ppms_search
 
   def provider_search(options = {})
     api.provider_locator(ppms_params.merge(options)).map do |provider|
-      prov_info = api.provider_info(provider['ProviderIdentifier'])
-      provider.add_details(prov_info)
+      begin
+        prov_info = api.provider_info(provider['ProviderIdentifier'])
+        provider.add_details(prov_info)
+      rescue => e
+        if Rails.env.production?
```

Comment:
you don't need the `Rails.env.production?` bit.  `log_exception_to_sentry` checks for `Settings.sentry.dsn.present?`